frontend/src/components/Dashboard/Opening/RecentOpeningsDataTable/testData.ts

```diff
@@ -0,0 +1,284 @@
+import { ITableHeader } from "../../../../types/TableHeader";
```

Comment:
What's this file for?

Reply:
this is for the headers for the table that is rendered, we dont have a unique on for each table yet. probably a good idea to create one for each.

Reply:
There are some hardcoded data down below. Where it is used? Is this some old, hardcoded data? If it is, please remove it. If is not, please reacess the necessity of keeping this specific data array on this file.

If this file is used to ser the data table header only, please rename it to something more meaningful and remove this hardcoded data, or move the hardcoded data to the test folder to be used by tests.